ui/apps.py

```diff
@@ -21,3 +21,7 @@ def ready(self):
         if missing_settings:
             raise ImproperlyConfigured(
                 'The following settings are missing: {}'.format(', '.join(missing_settings)))
+
+        # write the x509 certification & key to files
+        from ui.utils import write_x509_files
+        write_x509_files()
```

Comment:
It has something to do with deploying to heroku. Originally these keys were expected to already exist as files along with some others (like the cloudfront key).  But it wasn't possible to deploy these files to heroku so they were all changed to environment variables.  Most could be read directly without the need for a file to be written, but the x509 keys don't fall in that category because the mit-moira library uses `zeep` which in turn relies on `requests` which [only reads certificates from a file](http://docs.python-requests.org/en/master/user/advanced/#client-side-certificates).